src/player.js

```diff
@@ -25,6 +27,16 @@ module.exports.getSlackUser = function(users, message) {
     if (message.match[1]) {
         nameOrId = message.match[1];
     }
+    var player = getSlackUserFromNameOrID(users, nameOrId);
```

Comment:
:+1: 

Also, I the getPlayer method should probably exist in src/slack.js and the end users shouldn't have to pass in slack.users.  It should just use that directly. 

Thoughts?